### DISCUSSION TITLE
JsonNullable

### DISCUSSION THREAD

**Discussion by jbkervyn** [swagger-api/swagger-codegen#10986] [Q&A]
is there a support for the JsonNullable in spring generator (on the server side) ? like it is done in openapi-generator-maven-plugin?

see : https://github.com/OpenAPITools/jackson-databind-nullable

**Comment by akashlee** [comment]
nullable is deprecated in v3.1.0, so I don't think swagger codegen would take up this feature, but with minimal changes you could integrate jackson-databind-nullable with swagger codegen as I did.
https://stoplight.io/blog/difference-between-open-v2-v3-v31/
https://github.com/OAI/OpenAPI-Specification/tree/main/versions

1. Include jackson-databind-nullable in your project
2. Update model.mustache
`import org.openapitools.jackson.nullable.JsonNullable`
3. Update pojo.mustache file
```
{{#isNullable}}
  private {{{datatypeWithEnum}}} {{name}}{{#required}} = {{{defaultValue}}}{{/required}}{{^required}} = null{{/required}};
{{/isNullable}}

```
4. Declare the bean
```
    @Bean
    public Module jsonNullableModule() {
        return new JsonNullableModule();
    }

```
It should generate the members with JsonNullable such as 
`private JsonNullable<String> firstField = JsonNullable.undefined();`
